### DISCUSSION TITLE
Allow plugins to be a field for the unique identifier

### DISCUSSION THREAD

**Discussion by kringkaste** [engram-design/FeedMe#908] [Ideas]
Hi! It would be very nice, if a plugin can decide, if it is a possible field for the unique identifier. At the moment, the function fieldCanBeUniqueId($field) is responsible and only allows build in fields. Or did I miss something?

**Comment by bleepsandblops** [comment]
@kringkaste had the same issue; it works if you extend one of the allowed fields https://github.com/craftcms/feed-me/blob/489507a762e9a5c44b6f7b40fd9588f81ae79140/src/web/twig/variables/FeedMeVariable.php#L288

so you can write:
`class XXXModuleField extends PlainText`
instead of 
`class XXXModuleField extends Field`
and it will work

**Reply by gixxy22** [reply]
Is there a way to do this without modifying the core file?
